packages/iris-grid/src/IrisGrid.tsx

```diff
@@ -2453,6 +2453,29 @@ export class IrisGrid extends Component<IrisGridProps, IrisGridState> {
       this.focusRowInGrid(row);
       return;
     }
+
+    let cursorRow;
+    let cursorColumn;
+    if (this.grid) {
+      ({ cursorRow, cursorColumn } = this.grid.state);
+    }
```

Comment:
This should probably be a check of its own or an assertion maybe. If there's no grid, how are we here? Pretty sure it would mean rendering hasn't occurred yet

Reply:
At this point in the code, we are only concerned with whether cursorRow/cursorColumn exists. So if the grid is not rendered, then shouldn't we just default everything to the empty string?

Reply:
Are we suggesting we never render the gotoRow UI at all if grid is not rendered?

Reply:
`this.grid` is a ref to the Grid component rendered. The only reason it's ever not set is on the initial render. So realistically this is always set, but TS doesn't know that

Reply:
right, assertNotNull would be a better choice then